### ISSUE TITLE
Exception is thrown in Worksheets.Copy() if the sheet contains an image with hyperlink or embedded object

### ISSUE THREAD

**Issue by mcshorty** [EPPlusSoftware/EPPlus#659]
Hello,

version: 6.0.4

An Exception is thrown in Worksheets.Copy() if the sheet contains an image with hyperlink or embedded object.

Precondition:

case "embedded object"
- open new excel
- rename first sheet to "Table1"
- select "Table1" - insert - object - create from file - select a pdf file - ok
- save excel file as myFile.xlsx

case "image with hyperlink"
- open new excel
- rename first sheet to "Table1"
- select "Table1" - insert picture
- select picture - right mouse click - hyperlink - address https://www.google.de - ok
- save excel file as myFile.xlsx

Use following code to reproduce the error.

```
using (var package = new ExcelPackage("myFile.xlsx"))
{
	package.Workbook.Worksheets.Copy("Table1", "Table1 copy");
}
```

Best Regards
Rob

**Comment by JanKallman** [comment]
I'll have a look

**Comment by JanKallman** [comment]
Trying your code with one of my images (png) seems to work fine. Can you attach the workbook with your image, so I can see what is going wrong?
EPPlus currently have no support for embedded objects, so the relations and id's of the embedded object are not copied to the new worksheet. We will add support for this in a coming version, but it's not planned at this time.

**Comment by mcshorty** [comment]
Hi Jan,

I apologize describing an incorrect test case. 
The issue occurs, if the image contains a hyperlink.
I corrected the title of the issue and the description of the precondition of the case "image with hyperlink".

Thank you very much
Rob

**Comment by JanKallman** [comment]
Ok, I'll have another look.

**Comment by JanKallman** [comment]
This is fixed in version 5.8.11 and 6.0.5
